__tests__/cairo1.test.ts

```diff
@@ -204,6 +206,61 @@ describeIfDevnetSequencer('Cairo 1 Devnet', () => {
       expect(expected).toEqual(status);
     });
   });
+
+  describe('Cairo1 Account contract', () => {
+    const provider = getTestProvider() as SequencerProvider;
+    const account = getTestAccount(provider);
+    let accountC1: Account;
+
+    beforeAll(async () => {
+      // Deploy Cairo 1 Account
+      const priKey = stark.randomAddress();
+      const pubKey = ec.starkCurve.getStarkKey(priKey);
+
+      const calldata = { publicKey: pubKey };
+
+      // declare account
+      const declareAccount = await account.declare({
+        contract: compiledC1Account,
+        casm: compiledC1AccountCasm,
+      });
+      await account.waitForTransaction(declareAccount.transaction_hash);
+      const accountClassHash = declareAccount.class_hash;
+
+      // fund new account
+      const toBeAccountAddress = hash.calculateContractAddressFromHash(
+        pubKey,
+        accountClassHash,
+        calldata,
+        0
+      );
+      const devnetERC20Address =
+        '0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7';
+      const { transaction_hash } = await account.execute({
+        contractAddress: devnetERC20Address,
+        entrypoint: 'transfer',
+        calldata: {
+          recipient: toBeAccountAddress,
+          amount: cairo.uint256(1_000_000_000_000_000),
+        },
+      });
+      await account.waitForTransaction(transaction_hash);
+
+      // deploy account
+      accountC1 = new Account(provider, toBeAccountAddress, priKey);
+      const deployed = await accountC1.deploySelf({
+        classHash: accountClassHash,
+        constructorCalldata: calldata,
+        addressSalt: pubKey,
+      });
+      const receipt = await account.waitForTransaction(deployed.transaction_hash);
+      expect(receipt).toMatchSchemaRef('GetTransactionReceiptResponse');
+    });
+
+    test('deploy Cairo1 Account from Cairo0 Account', () => {
+      expect(accountC1 instanceof Account);
```

Comment:
```suggestion
      expect(accountC1).toBeInstanceOf(Account);
```
`expect` needs to be followed by a matcher method or it will always pass.

Reply:
684e9b713d2275866f3da1c141c86f33c3491f5f